src/app/components/app-version.tsx

```diff
@@ -16,9 +16,9 @@ const AppVersionLabel = forwardRef<HTMLSpanElement, AppVersionLabelProps>(
     <styled.span
       ref={ref}
       textStyle="mono.02"
-      marginRight="10px"
-      mb="-4px"
-      ml="space.02"
+      marginTop="auto"
```

Comment:
We can prob always change these to shorthand (`mt`) when come across them. I noticed panda recommends it. I've been trying to change `background` when I see it to `bg`. Obviously not critical to do here.

Reply:
Good idea, I'll do it as I can when making final changes.

Reply:
https://www.notion.so/trustmachines/Favour-Panda-shorthand-syntax-fa020e4a701e4fefb325e5cab1a8bc7d?pvs=4